### DISCUSSION TITLE
On/Off list for nodes

### DISCUSSION THREAD

**Discussion by ntdviet** [Comfy-Org/ComfyUI-Manager#81] [Ideas]
Hi @ltdrdata ,
my custom nodes list is slowly filled up with a lot of unnecessary nodes from different packs that I installed. Often times, just because of  1-2 useful nodes from a certain pack, I have to installe the full package including a lot of nodes I don't need.
Is it possible for a new feature that kind of list out all the installed nodes, then give us the option to switch them on/off so we don't have to see them all when doing "Add nodes" or node search. I have too many similar nodes for the same purposes because of different packs :'))

Or if you already have a better approach to solve this, please give me a hint.

**Comment by ntdviet** [comment]
Example: Look at this mess. The only one I use is CR Load Lora because it has a nice On/Off switch. I'd rather unload or turn off all the others

![ảnh](https://github.com/ltdrdata/ComfyUI-Manager/assets/54492570/775da057-fbb1-4fcc-af5a-b6f2440f8ef8)

**Reply by ltdrdata** [reply]
I do have some thoughts on how this could be implemented, but I'm not sure if implementing it within ComfyUI-Manager would be the best approach.

btw, Why don't you use bypass for turn off Lora?

**Reply by ntdviet** [reply]
Yeah, maybe it's better to be done at a lower level, when Comfy starts up.

>btw, Why don't you use bypass for turn off Lora?

Oh ok, never paid attention to that option. Thanks for the hint
